src/org/camunda/latera/bss/http/HTTPRestProcessor.groovy

```diff
@@ -155,12 +157,14 @@ class HTTPRestProcessor {
 
         if (params.path) {
           request.uri.path = absolutePath(params.path.toString())
-          params.remove('path')
         }
         if (params.query) {
           request.uri.query = params.query
-          params.remove('query')
         }
+
+        params.remove('query')
```

Comment:
What if query does not exist in params?

Reply:
![image](https://user-images.githubusercontent.com/33573380/85378356-21a32e00-b543-11ea-9346-ebbc4acb7731.png)